draft-ietf-tls-tls13.md

```diff
@@ -3488,6 +3503,9 @@ access_denied
 decode_error
 : A message could not be decoded because some field was out of the
   specified range or the length of the message was incorrect.
+  This alert is used for errors where the message does not conform
+  to the formal protocol syntax (e.g., a vector is specified as
+  having the range <2..255> but is encoded as having 0 length.)
```

Comment:
no, since also encoding with a vector length of 3 when the values in vector are uint16 is malformed, and so is encoding of a struct that has additional fields after the ones that are defined